docs/java/java-project.md

```diff
@@ -19,91 +19,91 @@ If you run into any issues when using the features below, you can contact us by
 
 ## Project management
 
-Project Management in VS Code for Java requires the [Java Dependency Viewer](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-dependency) extension. This extension has many features including creating projects as well as viewing the package structure of the project and its dependencies.
+Managing a project in VS Code requires the [Project Manager for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-dependency) extension. The extension helps manage class paths and dependencies, and create new projects, packages and classes.
 
-<a class="tutorial-install-extension-btn" href="vscode:extension/vscjava.vscode-java-dependency">Install the Java Dependency Viewer</a>
+<a class="tutorial-install-extension-btn" href="vscode:extension/vscjava.vscode-java-dependency">Install the Project Manager for Java</a>
 
-### Create project
+### Project view
 
-In addition to creating a project through Maven Archetype, you can also use the following command to create a simple Java project: **Java: Create Java Project** from the Command Palette.
+Project view provides a hierarchy view of your project and its dependencies, and also provides entry points for project management related tasks.
```

Comment:
the extension supports both `hierarchy view` and `flat view`

Reply:
Updated to address the comment